### PR TITLE
Release 3.6.0

### PR DESCRIPTION
**Pull request by repeatedly**
- Update fluentd
- Update bundled libraries
- Add fluent-plugin-systemd for non-Windows environment.
- Remove lsb-core from centos8
- Add TD_AGENT_LOG_FILE env for systemd

### CODE REVIEW

File: plugin_gems.rb [hunk 1]
```diff
@@ -44,17 +44,17 @@
 unless windows?
   download "rdkafka", "0.7.0"
 end
-download "fluent-plugin-kafka", "0.12.1"
+download "fluent-plugin-kafka", "0.12.2"
 unless td_agent_2?
-  download "elasticsearch", "6.8.0"
-  download "fluent-plugin-elasticsearch", "3.5.5"
+  download "elasticsearch", "6.8.1"
```
Comment (cosmo0920) [inline plugin_gems.rb]: :+1:

File: plugin_gems.rb [hunk 2]
```diff
@@ -44,17 +44,17 @@
...
+  download "fluent-plugin-elasticsearch", "3.8.0"
```
Comment (cosmo0920) [inline plugin_gems.rb]: Should we still use fluent-plugin-elasticsearch 3.8.0 in td-agent 3.6.0?
I want to recommend to bundle fluent-plugin-elasticsearch 4.0.3 in the next td-agent release.

File: plugin_gems.rb [hunk 3]
```diff
@@ -64,5 +64,5 @@
 if windows?
   download 'win32-eventlog', '0.6.7'
   download 'winevt_c', '0.6.1'
-  download 'fluent-plugin-windows-eventlog', '0.4.3'
+  download 'fluent-plugin-windows-eventlog', '0.4.4'
```
Comment (cosmo0920) [inline plugin_gems.rb]: Should we update fluent-plugin-windows-eventlog to v0.4.5 in the next td-agent release?

Comment (flurreN) [reply plugin_gems.rb]: > Should we update fluent-plugin-windows-eventlog to v0.4.5 in the next td-agent release?

Can you also update to winevt_c 0.7.0 ?

File: plugin_gems.rb [hunk 4]
```diff
@@ -64,6 +64,6 @@
 download "fluent-plugin-td-monitoring", "0.2.4"
 if windows?
   download 'win32-eventlog', '0.6.7'
-  download 'winevt_c', '0.6.1'
-  download 'fluent-plugin-windows-eventlog', '0.4.5'
+  download 'winevt_c', '0.7.0'
```
Comment (cosmo0920) [inline plugin_gems.rb]: winevt_c v0.7.0 is not compatible for fluent-plugin-windows-eventlog v0.4.6....
Should I release fluent-plugin-windows-eventlog v0.5.0?

Comment (cosmo0920) [reply plugin_gems.rb]: I've released fluent-plugin-windows-eventlog v0.5.0. Could you use it?

Comment (repeatedly) [reply plugin_gems.rb]: Thanks. Just updated.

**Comment by flurreN** [comment]
Is there an ETA for 3.6.0 to be released?

**Pull request merged**